test/weights.jl

```diff
@@ -15,13 +17,33 @@ wv = weights(w)
 @test sum(wv) == 6.0
 @test !isempty(wv)
 
+b  = trues(3)
+bv = weights(b)
+@test eltype(bv) == Bool
+@test length(bv) == 3
+@test values(bv) == b
+@test sum(bv)    == 3
+@test !isempty(bv)
+
+c  = [1, 4, 3, 5]; d = [4, 7, 18, 9]; e = [1, 2, -5, 3];
+s  = sparse(c, d, e)
```

Comment:
`c`, `d` and `e` are vectors which I used for constructing the sparse matrix `s`.